app/utils.py

```diff
@@ -19,6 +19,8 @@ def __init__(self, **kwargs):
 
         # Allow other params to be passed in exception
         self._kwargs = kwargs
+        if log_message == '' and self.message != '':
```

Comment:
Existing functionality did not print a log message if none was set. This means that the caller can set message or log_message and it will be printed out and passed back. 
Only if the caller sets both will they be different.